exercises/concept/squeaky-clean/.meta/config.json

```diff
@@ -4,7 +4,8 @@
   ],
   "contributors": [
     "sanderploegsma",
-    "manumafe98"
+    "manumafe98",
+    "jagdishdrp"
```

Comment:
This should be your Github handle. I'd also like to add mrDonoghue here since they first suggested this change in the forums.

Reply:
I am sorry for the oversight. I have added mrDonoghue and arranged the handles in alphabetical order, which was not maintained previously (Before this PR).